src/components/SelectDropdown/index.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React, { Component } from 'react'
+import Dropdown from '@components/Dropdown'
+import DropdownTrigger from '@components/DropdownTrigger'
+import Icon from '@components/Icon'
+import Text from '@components/Text'
+import Button from '@components/Button'
+import {
+  Wrapper,
+  DropdownBubble,
+  OptionsList,
+  OptionRow,
+  DropdownActions
+} from './styles'
+
+type Props = {
+  onApply: (selectedOptions: string[]) => {},
+  options: { value: string, text: string }[]
```

Comment:
Using `Array<…>` instead of `…[]` is more clear imo, that’s what we‘ve been using in the API 🤷‍♂️